go/oasis-node/cmd/registry/node/node.go

```diff
@@ -333,6 +344,60 @@ func doIsRegistered(cmd *cobra.Command, args []string) {
 		cmdCommon.EarlyLogAndExit(err)
 	}
 
+	conn, client := doConnect(cmd)
+	defer conn.Close()
+
+	nodes, err := client.GetNodes(context.Background(), consensus.HeightLatest)
+	if err != nil {
+		logger.Error("failed to query nodes",
+			"err", err,
+		)
+		os.Exit(1)
+	}
+
+	nodeIdentity := loadNodeIdentity()
+
+	for _, node := range nodes {
+		if node.ID.Equal(nodeIdentity.NodeSigner.Public()) {
+			fmt.Println("node is registered")
+			os.Exit(0)
+		}
+	}
+	fmt.Println("node is not registered")
+	os.Exit(1)
+}
+
+func doGenProveFreshness(cmd *cobra.Command, args []string) {
```

Comment:
Oh, we're trying to deprecate these, let's try not to add any new commands here. Instead those should belong to the CLI (will be moved into its own repo, currently here: https://github.com/oasisprotocol/oasis-sdk/tree/main/cli).

Reply:
Right, I forgot. Will remove cmd and  add it latter to the CLI repository.